text/0000-macro-expansion-for-macro-input.md

```diff
@@ -147,7 +147,7 @@ Notice that in the `quote!` output, the _argument_ to the new call to `string_le
 
 # Reference-level explanation
 
-Currently, the compiler does actually perform something similar to the loop described in th section on [expansion order](#macro-expansion-and-marking). We could 'just' augment the step that identifies potential macro calls to also inspect the otherwise unstructured token trees within macro arguments.
+Currently, the compiler does actually perform something similar to the loop described in the section on [expansion order](#macro-expansion-and-marking). We could 'just' augment the step that identifies potential macro calls to also inspect the otherwise unstructured token trees within macro arguments.
```

Comment:
That 'just' can be removed without losing anything.

Reply:
👍